tools/convj.py

```diff
@@ -6,37 +6,19 @@
 :Website: http://github.com/jiffyclub
```

Comment:
Huh. Hadn't look through these files before. This looks like code I get out of other modules.

Reply:
It's a mess. I have no idea what's going on in there but at least it's now a slightly tidier mess.